src/sagemaker/fw_utils.py

```diff
@@ -134,6 +134,14 @@
     "1.12.0",
 ]
 
+TORCH_DISTRIBUTED_SUPPORTED_FRAMEWORK_VERSIONS = [
+    "1.11.0"
```

Comment:
The documentation at line 307 says `torch-distributed` is available in the SageMaker Trainium (trn1) PyTorch Deep Learning Containers starting v1.11.0. But the validation seems to allow only 1.11 as the supported framework.
Let's make sure the two of them are in sync.

Reply:
Updated